server/pbench/bin/pbench-server-set-result-state

```diff
@@ -19,13 +19,13 @@ export CONFIG=/opt/pbench-server/lib/config/pbench-server.cfg
 . $dir/pbench-base.sh
 ###########################################################################
 
-results_full_path=$1
-tarball=$2
+hostnames_full_path=$1
```

Comment:
This script is only used with the *old* scripts and should *not* change. It should *not* be used by the new scripts at all.